Library/Homebrew/extend/blank.rb

```diff
@@ -1,7 +1,6 @@
+# typed: strict
 # frozen_string_literal: true
 
-require "concurrent/map"
```

Comment:
Is this an upstream change or our change?

Reply:
This is our change. In case it's helpful, the first [commit](https://github.com/Homebrew/brew/pull/16259/commits/b0338417c73c6446597ec93e24b3ce780ab13190) is from the ActiveSupport source, subsequent commits ( https://github.com/Homebrew/brew/pull/16259/commits/ccbb05d5aa14f971c882a4936b1db7e1befb12d3 in this case), are our updates.

Reply:
It looks like `Concurrent::Map` was added to solve the problem where `String#blank?` would throw an error if the encoding was not UTF-8 (https://github.com/rails/rails/issues/28953). I doubt we have to worry that much about concurrency here.

Reply:
Thanks both!